wordpress-mobile/WordPress-Android
# Issue/6085 add toolbar analytics

### Fix
Add analytics for editor toolbar usage tracking as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6085.

### Test
0. Update [WordPressEditor dependencies](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/build.gradle#L54) to include the following.
```compile ('com.github.wordpress-mobile.WordPress-Aztec-Android:aztec:develop-SNAPSHOT')``` and run `./gradlew build --refresh-dependencies`
1. Set breakpoints on editor [toolbar click listener methods](https://github.com/wordpress-mobile/WordPress-Android/blob/85b16c8c0f94215bfdc7a044bff956ef9b543e3d/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java#L343-L430).
2. Run app in debug mode.
3. Tap editor toolbar format buttons.
4. Notice app stopping at breakpoint(s).

#### Note
The ***branch will not build*** until https://github.com/wordpress-mobile/AztecEditor-Android/pull/392 is merged and the library version containing those changes is included in the [WordPressEditor dependencies](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/build.gradle#L54).
